lib/babel.js

```diff
@@ -74,8 +77,33 @@ if (inputSourceMap) {
 	options.inputSourceMap = JSON.parse(inputSourceMap.map);
 }
 
+// load up custom compilers
+process.argv.slice(3).forEach(function (arg) {
+	var match = /^--compilers=.+?:(.+)$/i.exec(arg);
```

Comment:
I used `--compilers` before really researching why that is what mocha used.
Turns out the only reason they use that over `--require` is for `mocha --watch` (they add `.coffee` to the list of extensions that should be watched).

That means, really only `--require` is needed until we add watch support ourselves.